_posts/2016-02-09-literacy.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title:  "On Students, Ilios, and Digital Literacy"
+date:   2016-02-09 15:00:00
+categories: news
+---
+One of the most challenging things in using a tool like Ilios actually has nothing to do with the tool at all, really. It is a problem with communication and digital literacy, not just among our faculty and educational staff, but (and most significantly) among our students as well.
+
+Let's face it: Ilios as a tool and a platform is pretty wickedly complex. Lots of moving parts, thousand (if not millions) of lines of code, dozens of dependencies....pretty much like any other contemporary application being used as a solution. Even so, most of the complications, user issues, and challenges we have seen coming to the core Ilios team aren't really about that.
+
+Since the start of the year, the overwhelming majority of issues that have been reported to us around student use of Ilios have revolved around issues of general digital literacy and communication. In some cases, schools and programs have vastly overestimated our students' native understanding of the tools they use regularly -- their mobile phones, laptop computers, and wi-fi networks, for instance -- and made assumptions on top of that misunderstanding about how much guidance and orientation they may need to access networks and information here on campus. In other cases, schools and departments have neglected to provide them with even the barest outline of a process by which they can discover the sort of information they need to move ahead: where to go for technical assistance, what to do when they encounter problems accessing information on the web, in their online learning environments, in forums, or getting to a download or streaming presentation.
+
+This is disheartening. But what can we do about it?
+
+__Step #1: Better Communication__
+
+Even before we starting handing tools to students to help them in their study, we need to be sure that we are telling them clearly not just how to use the tools, but what those tools even are, and how they fit into the landscape of technology and services they need to add to their experience to be successful. That information should start at day one, and must be both consistent in content and over time. The technology ecosystem we work in is complex; with the rapid proliferation of new tools and processes, changes in social media use and asynchronous learning models, it is only getting more complicated and dense. It is incumbent on us, as educational technologists, to effectively communicate this complexity to our communities. This is easy to say: it is hard to do. Nevertheless, without effective communications about the stuff that makes our stuff work -- the underpinnings of our systems, rather than the tools themselves -- any challenges faced by our users will wind up being both misdirected and ultimately dissatisfied with the solutions they are offered.

```

Comment:
suggesting replacing "about the stuff that makes our stuff work" with "about the things that makes our stuff work".